app/assets/javascripts/admin/addon/controllers/admin-users-list-show.js

```diff
@@ -106,4 +116,42 @@ export default class AdminUsersListShowController extends Controller.extend(
       asc,
     });
   }
+
+  @action
+  toggleBulkSelect() {
+    this.bulkSelect = !this.bulkSelect;
+    this.displayBulkActions = false;
+    this.bulkSelectedUsersMap = {};
+    this.bulkSelectedUserIdsSet = new Set();
+  }
+
+  @action
+  bulkSelectItemToggle(userId, event) {
```

Comment:
Would be good to enforce the 100 user max here as well, to save a trip to the server for the user if they click more than 100. Just pop a toast to say the max amount of users for bulk actions has been reached

Reply:
done